frontend/packages/shared/src/contexts/ServicesContext.tsx

```diff
@@ -39,25 +40,35 @@ const handleError = (
 ): void => {
   // TODO : log axios errors
 
-  if (error?.response?.status === ServerCodes.Unauthorized) {
-    const errorMessageKey = 'api_errors.Unauthorized';
+  const renderToast = (key: string, options: ToastOptions = {}) => {
+    const errorMessageKey = `api_errors.${key}`;
     if (i18n.exists(errorMessageKey)) {
-      toast.error(t(errorMessageKey), { toastId: errorMessageKey, autoClose: LOG_OUT_TIMER_MS });
+      toast.error(t(errorMessageKey), {
+        toastId: errorMessageKey,
+        ...options,
+      });
+    } else {
+      toast.error(t('general.error_message'), {
+        toastId: errorMessageKey,
+        ...options,
+      });
```

Comment:
The `general.error_message` contains a variable that must be converted to a link.
A solution can be to create a new method for the default error message, something like:

```
const renderDefaultToast = () => {
  toast.error(
    () => (
      <Trans
        i18nKey={'general.error_message'}
        components={{
          a: (
            <Link href='/contact' inverted={true}>
              {' '}
            </Link>
          ),
        }}
      />
    ),
    { toastId: 'default' },
  );
};
```

And then call it like this:

```suggestion
      renderDefaultToast();
```